### PR TITLE
(fix): change gha to target main

### PR DESCRIPTION
**Pull request by AjayJagan**
## Description
Change the GH actions for release process to target main branch.

## Merge criteria

- [ ] You have read the [contributors guide](https://github.com/opendatahub-io/opendatahub-operator/blob/incubation/CONTRIBUTING.md).
- [ ] Commit messages are meaningful - have a clear and concise summary and detailed explanation of what was changed and why.
- [ ] Pull Request contains a description of the solution, a link to the JIRA issue, and to any dependent or related Pull Request.
- [ ] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [ ] The developer has manually tested the changes and verified that the changes work

### CODE REVIEW

**Comment by AjayJagan** [comment]
@VaishnaviHire ^

**Comment by biswassri** [comment]
this looks good to me, but I'll let @VaishnaviHire review

**Review by asanzgom — APPROVED** [review]

File: .github/workflows/pre-release.yaml [hunk 1]
```diff
@@ -38,19 +38,19 @@ jobs:
...
-        base-branch: incubation
+        base-branch: main
     - name: Create release branch
       run: |
-        git fetch origin incubation:incubation
-        git checkout incubation
+        git fetch origin main:main
+        git checkout main
         git checkout -b odh-${{ env.VERSION }}
         git push -f origin odh-${{ env.VERSION }}
```
[the remark this answers is not in the document]
Comment (AjayJagan) [reply .github/workflows/pre-release.yaml:55]: yes it could be done. I can do that in a separate pr. Thanks for the idea @ykaliuta

**Pull request merged**